blackbox/docs/general/builtins/scalar.rst

```diff
@@ -2338,6 +2338,32 @@ Example::
     +--------------------+
     SELECT 1 row in set (... sec)
 
+.. _pg_typeof:
+
+``pg_typeof(any)``
+==================
+
+The function ``pg_typeof`` returns the text representation of the value's data
+type passed to it.
+
+Returns: ``text``
+
+Synopsis::
+
+   pg_typeof(any)
```

Comment:
```suggestion
   pg_typeof(expression)
```

Reply:
ah!!, thank you

Reply:
hm it is still `(any)`

Reply:
Apologies, I misunderstood, I did not realise the title had the parenthesis containing the expression, while other function titles did not. Corrected.